ledger/participant-integration-api/src/main/scala/platform/store/dao/events/PostCommitValidation.scala

```diff
@@ -113,8 +114,13 @@ private[dao] object PostCommitValidation {
       def foldInformees[T](tx: CommittedTransaction, init: T)(
           f: (T, Party) => T
       ): T =
-        tx.fold(init) { case (accum, (_, node)) =>
-          node.informeesOfNode.foldLeft(accum)(f)
+        tx.fold(init) {
+          case (accum, (_, node: LFNode.GenActionNode[_, _])) =>
+            node.informeesOfNode.foldLeft(accum)(f)
+          case (accum, (_, _: LFNode.NodeRollback[_])) =>
+            // TODO https://github.com/digital-asset/daml/issues/8020
```

Comment:
and here

Reply:
also here.

Reply:
Duplicate of the one above for non-appendonly so same answer.